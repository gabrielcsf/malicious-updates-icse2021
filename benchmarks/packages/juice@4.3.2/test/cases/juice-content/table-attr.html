<html>
<head>
    <style>
        p, td, img {
            text-align: center;
            vertical-align: middle;
        }

        table {
            background-image: url('test.png');
            background-color: red;
        }

        td.double {
            background-image: url("test.png");
        }

        td.single {
            background-image: url('test.png');
        }

        td.none {
            background-image: url(test.png);
        }
    </style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <td class="double">
                wide
            </td>
            <td class="single">
                wide
            </td>
            <td class="none">
                wide
            </td>
        </tr>
    </table>
</body>
</html>
