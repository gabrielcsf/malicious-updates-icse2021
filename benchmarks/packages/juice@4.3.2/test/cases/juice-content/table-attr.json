{
    "url": "./",
    "removeStyleTags": true,
    "applyAttributesTableElements": true
}
