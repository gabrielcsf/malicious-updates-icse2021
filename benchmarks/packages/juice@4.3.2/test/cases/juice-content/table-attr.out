<html>
<head>
    
</head>
<body>
    <p style="text-align: center; vertical-align: middle;">none</p>
    <table style="background-image: url('test.png'); background-color: red;" background="test.png" bgcolor="red">
        <tr>
            <td class="double" style="text-align: center; vertical-align: middle; background-image: url('test.png');" align="center" valign="middle" background="test.png">
                wide
            </td>
            <td class="single" style="text-align: center; vertical-align: middle; background-image: url('test.png');" align="center" valign="middle" background="test.png">
                wide
            </td>
            <td class="none" style="text-align: center; vertical-align: middle; background-image: url(test.png);" align="center" valign="middle" background="test.png">
                wide
            </td>
        </tr>
    </table>
</body>
</html>
